tasks/us/census/tiger.py

```diff
@@ -663,15 +663,18 @@ def timespan(self):
 
     def populate(self):
         session = current_session()
-        stmt = ('INSERT INTO {output} '
-                'SELECT geoid, ST_Union(ST_MakePolygon(ST_ExteriorRing(the_geom))) AS the_geom, '
-                '       MAX(aland) AS aland '
-                'FROM ( '
-                '    SELECT geoid, (ST_Dump(the_geom)).geom AS the_geom, '
-                '           aland '
-                '    FROM {input} '
-                ") holes WHERE GeometryType(the_geom) = 'POLYGON' "
-                'GROUP BY geoid'.format(
+
+        stmt = ('''INSERT INTO {output}
+                   SELECT
+                     geoid,
+                     ST_Union(ARRAY(
+                       SELECT ST_MakePolygon(ST_ExteriorRing(
+                         (ST_Dump(the_geom)).geom
+                       ))
+                       WHERE GeometryType(the_geom) = 'POLYGON'
+                     )),
+                     aland
+                   FROM {input}'''.format(
```

Comment:
Both queries return the same results (checked with states and counties). Analyzing the queries with `EXPLAIN ANALYZE`, the query planner is using a `HashAggregate` for the old query and a `Seq Scan` for the new query, so the new query is working as expected (performing the memory consuming operations per row instead of storing the whole table in memory)